com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/analytics/collectors/AnalyticsCollector.java

```diff
@@ -18,8 +18,9 @@
 import com.sap.cloud.lm.sl.cf.process.Constants;
 import com.sap.cloud.lm.sl.cf.process.analytics.model.AbstractCommonProcessAttributes;
 import com.sap.cloud.lm.sl.cf.process.analytics.model.AnalyticsData;
-import com.sap.cloud.lm.sl.cf.process.steps.StepsUtil;
 import com.sap.cloud.lm.sl.cf.process.util.ProcessTypeParser;
+import com.sap.cloud.lm.sl.cf.process.variables.VariableHandling;

```

Comment:
In the first commit. I decided to make it static, since in a lot of listeners, I had to create new instances of `VariableHandler` in multiple places and it just did not seem like an improvement.